files/zh-tw/web/javascript/reference/operators/addition/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: 相加運算子 (+)
+slug: Web/JavaScript/Reference/Operators/Addition
+tags:
+  - JavaScript
+  - Language feature
+  - Operator
+  - Reference
+browser-compat: javascript.operators.addition

```

Comment:
Yes, we can just remove it (and it's also ok to keep it, but we need to modify it when it is modified in mdn/content)

Reply:
The `browser-compat` metadata is used by `{{Compat}}` and `{{Specifications}}` macros, and it can be merged from `en-US`:

https://github.com/mdn/yari/blob/546cb5a7736167f4a805be5089964c703ee630d3/kumascript/index.js#L58-L69